docs/guides/setup/configuring_data_contexts/how_to_configure_credentials_using_a_yaml_file_or_environment_variables.md

```diff
@@ -61,15 +61,15 @@ Steps
 2. If using a YAML file, set the ``config_variables_file_path`` key in your ``great_expectations.yml`` or leave the default.
 
   ```yaml
-  # great_expectations/great_expectations.yml
+  # great-expectations/great_expectations.yml
 
   config_variables_file_path: uncommitted/config_variables.yml
   ```
 
 3. Replace credentials or other values in your ``great_expectations.yml`` with ``${}``-wrapped variable names (i.e. ``${ENVIRONMENT_VARIABLE}`` or ``${YAML_KEY}``).
 
   ```yaml
-  # great_expectations/great_expectations.yml
+  # great-expectations/great_expectations.yml
```

Comment:
This does not seem to be correct -- should this not remain using the underscore?  If agree, then please verify all similar cases (and must be different from the location on GitHub where the hyphen is used).

Reply:
Changed this as well. Great catch, thank you so much!